external-crates/move/crates/move-compiler/tests/development/enums/matching/invalid_empty_ctor.exp

```diff
@@ -2,21 +2,21 @@ error[E03013]: positional call mismatch
   ┌─ tests/development/enums/matching/invalid_empty_ctor.move:8:18
   │
 4 │         None
-  │         ---- Variant is defined here.
+  │         ---- Enum variant is declared here
```

Comment:
Thoughts on possibly having this be "'None' is declared here" ? Don't have super strong opinions on it though.

Reply:
It took a little plumbing, but done!